src/components/new-safe/steps/Step1/index.tsx

```diff
@@ -0,0 +1,127 @@
+import {
+  Select,
+  InputAdornment,
+  InputLabel,
+  MenuItem,
+  TextField,
+  Tooltip,
+  SvgIcon,
+  Typography,
+  Button,
+  Link,
+} from '@mui/material'
+import { useForm } from 'react-hook-form'
+
+import ChainIndicator from '@/components/common/ChainIndicator'
+import useChainId from '@/hooks/useChainId'
+import useChains from '@/hooks/useChains'
+import { useMnemonicSafeName } from '@/hooks/useMnemonicName'
+import InfoIcon from '@/public/images/notifications/info.svg'
+import StepCard from '../../StepCard'
+
+import css from './styles.module.css'
+
+type CreateSafeStep1Form = {
+  chainId: string
+  name: string
+}
+
+enum CreateSafeStep1Fields {
+  chainId = 'chainId',
+  name = 'name',
+}
+
+const STEP_1_FORM_ID = 'create-safe-step-1-form'
+
+const CreateSafeStep1 = () => {
+  const chainId = useChainId()
+  const { configs } = useChains()
+  const fallbackName = useMnemonicSafeName()
+
+  const {
+    handleSubmit,
+    register,
+    formState: { errors },
+  } = useForm<CreateSafeStep1Form>({
+    mode: 'all',
+    defaultValues: {
+      [CreateSafeStep1Fields.chainId]: chainId || '',
+      [CreateSafeStep1Fields.name]: '',
+    },
+  })
+
+  const onSubmit = (data: CreateSafeStep1Form) => {
+    console.log(data)
+  }
+
+  return (
+    <StepCard
+      title="Select network and name Safe"
+      subheader="Select the network on which to create your Safe"
+      content={
+        <form onSubmit={handleSubmit(onSubmit)} id={STEP_1_FORM_ID} className={css.form}>
+          <Select
+            startAdornment={
+              <InputAdornment position="start">
+                <InputLabel>Network</InputLabel>
+              </InputAdornment>
+            }
+            {...register(CreateSafeStep1Fields.chainId)}
+            SelectDisplayProps={{
+              className: css.select,
+            }}
+            defaultValue={chainId}
+          >
+            {configs.map((chain) => (
+              <MenuItem key={chain.chainId} value={chain.chainId}>
+                <ChainIndicator chainId={chain.chainId} className={css.chain} />
+              </MenuItem>
+            ))}
+          </Select>
```

Comment:
The Menu items look different than the network selector in the header. Could we reuse `NetworkSelector` here? The main difference seems to be the label "Network" in front.

<img width="177" alt="Screenshot 2022-10-25 at 11 42 50" src="https://user-images.githubusercontent.com/5880855/197740681-21f428f4-ceaa-4e24-abcd-4c1b7515bf3d.png">